src/features/user/MFV2/ForestProgress/microComponents/PersonalPercentage.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from '../ForestProgress.module.scss';
+import { BarsProps } from './Bars';
+
+const PersonalPercentageBar = ({
+  personalPercentage,
+  giftPercentage,
+  target,
+  gift,
+  personal,
+}: BarsProps) => {
+  const totalAchievment = gift + personal;
+  return (
+    <div
+      style={{
+        width: `${personalPercentage}%`,
+        borderTopRightRadius: `${
+          giftPercentage !== 0 || target > totalAchievment ? 0 : 5
+        }px`,
+        borderBottomRightRadius: `${
+          giftPercentage !== 0 || target > totalAchievment ? 0 : 5
+        }px`,
+      }}
```

Comment:
You don't really need to programatically provide a border radius for `PersonalPercentage` and `GiftPercentage`. Having border-radius on the parent `barContainer` div with`overflow: hidden` will take care of the border radius.